Docs/EnviromnentSetup/Unittest_setup.md

```diff
@@ -0,0 +1,98 @@
+# Data Factory - Testing Framework
+
+## Setup 
+
+This is an extra step for setup in case you need additional help to do it. The Unit Test does not need to rigorously  follow these steps to work. You just need to start your Python project, choose your testing library, and start to code. Though, if you need additional assistance how to do it which library to use, and some code examples. Feel free to use this guide. 
+
+### Step by Step 
+
+If for example you are using Visual Studio Code:
+
+For more references: [Get Started Tutorial for Python in Visual Studio Code](https://code.visualstudio.com/docs/python/python-tutorial)
+
+1. Open your new Python project. 
+![image](https://github.com/LiliamLeme/data-factory-testing-framework/assets/62876278/9f76474d-b365-43e5-8fb0-dc074cdb1584)
```

Comment:
Can you host these assets under docs/assets in this repository so we contain everything in the same repo?